manifests/o/ONLYOFFICE/DesktopEditors/7.3.3.99/ONLYOFFICE.DesktopEditors.locale.en-US.yaml

```diff
@@ -0,0 +1,38 @@
+# Created with WinGet Releaser v2 using Komac v1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ONLYOFFICE.DesktopEditors
+PackageVersion: 7.3.3.99
+PackageLocale: en-US
+Publisher: Ascensio System SIA
+PublisherUrl: https://github.com/ONLYOFFICE
+PublisherSupportUrl: https://github.com/ONLYOFFICE/DesktopEditors/issues
+Author: Ascensio System SIA
+PackageName: ONLYOFFICE Desktop Editors
+PackageUrl: https://www.onlyoffice.com/en/desktop.aspx
+License: AGPL-3.0
+LicenseUrl: https://github.com/ONLYOFFICE/DesktopEditors/blob/master/LICENSE
+CopyrightUrl: https://github.com/ONLYOFFICE/DesktopEditors/blob/master/LICENSE
+ShortDescription: All the tools you need to work with text documents, spreadsheets, presentations, and fillable forms on Windows, Linux, and macOS
+Description: |-
+  Create, view and edit text documents, spreadsheets and presentations of any size and complexity.
+  Work on documents of most popular formats; DOCX, ODT, XLSX, ODS, CSV, PPTX, ODP, etc.
+  Deal with multiple files within one and the same window thanks to the tab-based user interface.
+Moniker: onlyoffice
+Tags:
+- document
+- editor
+- excel
+- office
+- powerpoint
+- presentation
+- spreadsheet
+- text-editor
+- word
+ReleaseNotesUrl: https://github.com/ONLYOFFICE/DesktopEditors/releases/tag/v7.3.3.99

```

Comment:
The actual url is https://github.com/ONLYOFFICE/DesktopEditors/releases/tag/v7.3.3